src/Cipher.php

```diff
@@ -1,22 +1,28 @@
 <?php
 
-namespace Athens\Encryption;
-
 /**
- * Class Cipher
- *
- * Singleton class encapsulating encryption/decryption of data fields
- *
- * @package Athens\Encryption
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
  */
+
+namespace Spryker\PropelEncryptionBehavior;
+
+use Exception;
+
 class Cipher
 {
-    const IV_SIZE = 16;
+    /**
+     * @var int
+     */
+    public const IV_SIZE = 16;
```

Comment:
What is IV? Can you use a more speaking name for it?

Reply:
It comes from the original repository

Reply:
IV stands for initialization vector and IV is a common abbreviation https://en.wikipedia.org/wiki/Initialization_vector

Reply:
We usually do not use abbreviations. I would rename.

Reply:
He already did :)

Reply:
It is already renamed